CMakeLists.txt

```diff
@@ -617,6 +637,8 @@ if(NOT ANDROID)
 		set(PORTAUDIO_LIBRARIES portaudio)
 	else()
 		message(STATUS "Using static PortAudio from Externals for mic support")
+		set(PA_PRIVATE_COMPILE_DEFINITIONS PAWIN_WDMKS_NO_KSGUID_LIB)
+		set(PA_USE_WDMKS OFF CACHE BOOL "" FORCE)
 		add_subdirectory(Externals/portaudio)
 		set(PORTAUDIO_LIBRARIES portaudio_static)
```

Comment:
It's fine if you update in your other PR, I'll update whatever is needed for Windows CMake later.